src/Native/Runtime/MiscHelpers.cpp

```diff
@@ -749,3 +749,10 @@ COOP_PINVOKE_HELPER(void, RhGetCurrentThreadStackBounds, (PTR_VOID * ppStackLow,
 {
     ThreadStore::GetCurrentThread()->GetStackBounds(ppStackLow, ppStackHigh);
 }
+
+ThreadExitCallback g_threadExitCallback;
```

Comment:
@jkotas I think the callback may be useful on Windows as well.  We might even try to merge Unix and Windows implementations.